### DISCUSSION TITLE
Many errors WalletConnect

### DISCUSSION THREAD

**Discussion by Xaoc13-web** [WalletConnect/walletconnect-monorepo#4620] [Developer Support]
Hey. 
I set up a server, thought everything was good, but then I discovered that WalletConnect isn’t working. I didn’t write the script, so it’s very difficult to figure out, especially since I’m also a newbie in web3. Here’s a screenshot of the errors.
![Снимок экрана (107)](https://github.com/WalletConnect/walletconnect-monorepo/assets/67403709/c4b0d48d-98f8-44c1-a759-e486c2d2f48e)

**Comment by glitch-txs** [comment]
That's v1 it's outdated and doesn't work any more, you should update to v2

https://docs.walletconnect.com/appkit/overview

**Comment by Xaoc13-web** [comment]
What should I do to upgrade to version v2?
Change the code or files? or both?

**Reply by glitch-txs** [reply]
You need to remove everything and start over from scratch because that's a very very old version

**Comment by Xaoc13-web** [comment]
I contacted the person who wrote all this, and he said that everything should work. I am completely confused and don’t know what to do with all this

**Reply by glitch-txs** [reply]
We deprecated that version at the beginning of last year, it stopped working on a specific date that we announced.
https://medium.com/walletconnect/walletconnect-v1-0-shutdown-what-to-expect-and-how-to-prepare-2ae443d6d7bd
